extensions/wikia/Chat2/resourceloader/ResourceLoaderChatSiteModule.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+/**
+ * Resource Loader module for site JS and CSS for inclusion in Special:Chat
+ */
+class ResourceLoaderChatSiteModule extends ResourceLoaderSiteModule {
+
+	protected function getPages( ResourceLoaderContext $context ) {
+		$pages = [
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Immediately return this expression instead of assigning it to the temporary variable "$pages". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1488)